experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/PostDeployScripts/prepareSrc.cmd

```diff
@@ -0,0 +1,89 @@
+@echo off
+setlocal
+SET password=%1
+SET exportSettings=%2
+SET repoName=srcRepo
+SET repoUrl=file:///%HOMEDRIVE:~0,1%/%HOMEPATH:~1%/site/%repoName%
```

Comment:
This won't work on Mac or Linux. 

Please remove and consider an alternate solution.